### ISSUE TITLE
Do a web accessibility  for WCAG compliance

### ISSUE THREAD

**Issue by ramiz4** [ramiz4/ramiz-loki#30]
https://ramiz4.github.io/ramiz-loki/
We found 22 accessibility violations on your page.

These issues leave your business at risk of facing legal action. Take action to protect your business, and review your results to get started.

Find all issues and fix them!
